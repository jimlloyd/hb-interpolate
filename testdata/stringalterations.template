original: {{description}}
Capitalize: {{Capitalize description}}
CamelCase: {{CamelCase description}}
camelCase: {{camelCase description}}
snake_case: {{snake_case description}}
lowercase: {{lowercase description}}
UPPERCASE: {{UPPERCASE description}}
lowerFirst: {{lowerFirst description}}
UpperFirst: {{UpperFirst description}}
kebab-case: {{kebab-case description}}
