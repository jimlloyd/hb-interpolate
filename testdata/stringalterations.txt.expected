original: Interpolate a json file through a Handlebars template file to stdout.
Capitalize: Interpolate a json file through a handlebars template file to stdout.
CamelCase: InterpolateAJsonFileThroughAHandlebarsTemplateFileToStdout
camelCase: interpolateAJsonFileThroughAHandlebarsTemplateFileToStdout
snake_case: interpolate_a_json_file_through_a_handlebars_template_file_to_stdout
lowercase: interpolate a json file through a handlebars template file to stdout
UPPERCASE: INTERPOLATE A JSON FILE THROUGH A HANDLEBARS TEMPLATE FILE TO STDOUT.
lowerFirst: interpolate a json file through a Handlebars template file to stdout.
UpperFirst: Interpolate a json file through a Handlebars template file to stdout.
kebab-case: interpolate-a-json-file-through-a-handlebars-template-file-to-stdout
